persist/schema/types.go

```diff
@@ -20,19 +20,25 @@
 
 package schema
 
+import (
+	"github.com/m3db/m3db/persist"
+)
+
 // MajorVersion is the major schema version for a set of fileset files,
 // this is only incremented when breaking changes are introduced and
 // tooling needs to upgrade older files to newer files before a server restart
 const MajorVersion = 1
 
 // IndexInfo stores metadata information about block filesets
 type IndexInfo struct {
-	Start        int64
+	BlockStart   int64
 	BlockSize    int64
 	Entries      int64
 	MajorVersion int64
 	Summaries    IndexSummariesInfo
 	BloomFilter  IndexBloomFilterInfo
+	SnapshotTime int64
```

Comment:
more a question: do you need this anymore? What does this give you over picking the latest index?

Reply:
I need this so I can accurately determine how much of the commit log I need to read.

I.E if I have three sets of snapshot files on disk, I just pick the one with the highest index to read, but once I've picked that one I need to know what moment in time the snapshot file captures "up to" so I can determine how much of the commit log files I need to bootstrap

Reply:
fair, I forgot about that.